src/styles/card.css

```
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+  font-family: "Roboto", serif;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 3px;
+}
+
+.card__buy-button {
+  display: flex;
+  width: 166px;
```

do not hardcode width 
```suggestion
  width: 100%;
```